pkg/securitycontextconstraints/sccmatching/matcher.go

```diff
@@ -196,21 +224,12 @@ func CreateProvidersFromConstraints(ns string, sccs []*securityv1.SecurityContex
 }
 
 // CreateProviderFromConstraint creates a SecurityContextConstraintProvider from a SecurityContextConstraint
-func CreateProviderFromConstraint(ns string, namespace *corev1.Namespace, constraint *securityv1.SecurityContextConstraints, client kubernetes.Interface) (SecurityContextConstraintsProvider, *corev1.Namespace, error) {
+func CreateProviderFromConstraint(namespace *corev1.Namespace, constraint *securityv1.SecurityContextConstraints) (SecurityContextConstraintsProvider, error) {
 	var err error
 	resolveUIDRange := requiresPreAllocatedUIDRange(constraint)
 	resolveSELinuxLevel := requiresPreAllocatedSELinuxLevel(constraint)
 	resolveFSGroup := requiresPreallocatedFSGroup(constraint)
 	resolveSupplementalGroups := requiresPreallocatedSupplementalGroups(constraint)
-	requiresNamespaceAllocations := resolveUIDRange || resolveSELinuxLevel || resolveFSGroup || resolveSupplementalGroups
```

Comment:
nit: we could move the above functions to the place of use now

Reply:
fixed